main/tests/cypress/cypress.json

```diff
@@ -2,12 +2,11 @@
   "integrationFolder": "./cypress/integration",
   "nodeVersion": "system",
   "retries": {
-    "runMode": 3,
+    "runMode": 4,
```

Comment:
Isnt this too much to retry like a test is buggy if it is failing for 3 times, dont you think so 😅 ?